backend/rs/ic_vetkeys/src/utils/mod.rs

```diff
@@ -589,6 +586,36 @@ impl IBECiphertext {
     }
 }
 
+/// Verify an augmented BLS signature
```

Comment:
It might make sense to give more details on what "augmented" means. So maybe turn "augmented" into a link to
* https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature#name-message-augmentation-2 or
* https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature#name-message-augmentation

Reply:
Done